project.clj

```diff
@@ -7,6 +7,26 @@
     :password :env/nexus_jenkins_password
     :sign-releases false })
 
+(def heap-size-from-profile-clj
+  (let [profile-clj (io/file (System/getenv "HOME") ".lein" "profiles.clj")]
+    (if (.exists profile-clj)
+      (-> profile-clj
+        slurp
+        read-string
+        (get-in [:user :puppetserver-heap-size])))))
+
+(defn heap-size
+  [default-heap-size heap-size-type]
+  (or
+    (System/getenv "PUPPETSERVER_HEAP_SIZE")
+    heap-size-from-profile-clj
+    (do
+      (println "Using" default-heap-size heap-size-type
```

Comment:
I'm not sure there's a way to avoid it.  I see those messages if I don't have the settings defined even if I move the `:jvm-opts` section down into the `:test` profile and just run `lein version`.  @cprice404 - do you have any ideas on that?